Source/Editor/DualityEditor/Controls/AutoExpandTreeView.cs

```diff
@@ -0,0 +1,68 @@
+using Aga.Controls.Tree;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Duality.Editor.Controls
+{
+	public class AutoExpandTreeView : TreeViewAdv
+	{
+		public int DragOverExpandDelay
```

Comment:
Looking at this a bit more, it's really just a property exposing the the Interval property of the timer. Do code guidelines prefer just doing the following:

```
private Timer       autoExpandDelayTimer = new Timer();
public int AutoExpandDelay
{
    get
    {
        return autoExpandDelayTimer.Interval;
    }

    set
    {
        autoExpandDelayTimer.Interval = value;
    }
}
```